code/datums/mind.dm

```diff
@@ -73,6 +73,16 @@
 
 /datum/mind/Destroy()
 	SSticker.minds -= src
+	//Clear references
+	for(var/i in GLOB.mob_living_list)

```

Comment:
As said before the only natural occurrence here should be new_player minds.  Did you find some scenario where it happens ?

Reply:
on server i'm not actualyl sure if this happens as i obviously can't run find_references there, but locally i believe it's happened otherwise i probably wouldn't have caught it.

Reply:
I'd prefer if you found the cause of this first then.